drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -443,8 +441,63 @@ function DragAndDropTemplates(configuration) {
         )
     };
 
+    var progressTemplate = function(ctx) {
+        // Formats a number to 4 decimals without trailing zeros
+        // (1.00 -> '1'; 1.50 -> '1.5'; 1.333333333 -> '1.3333').
+        var formatNumber = function(n) { return n.toFixed(4).replace(/\.?0+$/, ''); };
+        var is_graded = ctx.graded && ctx.weighted_max_score > 0;
+        var progress_template;
+        if (ctx.grade !== null && ctx.weighted_max_score > 0) {
+            if (is_graded) {
+                progress_template = ngettext(
+                      // Translators: {earned} is the number of points earned. {possible} is the total number of points (examples: 0/1, 1/1, 2/3, 5/10). The total number of points will always be at least 1. We pluralize based on the total number of points (example: 0/1 point; 1/2 points).
+                    '{earned}/{possible} point (graded)',
+                    '{earned}/{possible} points (graded)',
+                    ctx.weighted_max_score
+                );
+            } else {
+                progress_template = ngettext(
+                      // Translators: {earned} is the number of points earned. {possible} is the total number of points (examples: 0/1, 1/1, 2/3, 5/10). The total number of points will always be at least 1. We pluralize based on the total number of points (example: 0/1 point; 1/2 points).
+                    '{earned}/{possible} point (ungraded)',
+                    '{earned}/{possible} points (ungraded)',
+                    ctx.weighted_max_score
+                );
+            }
+            progress_template = progress_template.replace('{earned}', formatNumber(ctx.grade));
+        } else {
```

Comment:
@sstack22 I believe this is a product decision, so please decide and create a task in JIRA if modifications are needed.